src/sentry/static/sentry/app/components/sidebar/broadcasts.tsx

```diff
@@ -1,33 +1,41 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 
 import {getAllBroadcasts, markBroadcastsAsSeen} from 'app/actionCreators/broadcasts';
 import {t} from 'app/locale';
 import LoadingIndicator from 'app/components/loadingIndicator';
-import SentryTypes from 'app/sentryTypes';
 import SidebarItem from 'app/components/sidebar/sidebarItem';
 import SidebarPanel from 'app/components/sidebar/sidebarPanel';
 import SidebarPanelEmpty from 'app/components/sidebar/sidebarPanelEmpty';
 import SidebarPanelItem from 'app/components/sidebar/sidebarPanelItem';
 import {IconBroadcast} from 'app/icons/iconBroadcast';
 import withApi from 'app/utils/withApi';
+import {Client} from 'app/api';
+import {Organization, Broadcast} from 'app/types';
+
+import {SidebarOrientation, SidebarPanelKey} from './types';
```

Comment:
Is there a reason for the empty line (line 14)
Is there any rules around how to order the imports?

Reply:
Yeah we have eslint rules for this, the reason is because `./types` is a relative import, where as the ones before are absolute.